petpals/messaging/templates/messages.html

```diff
@@ -28,21 +28,33 @@ <h1>Messages</h1>
                         <a href="messages/conversation/{{ message.conversation_id }}" class="message-link">
                             <div class="message-card-body">
                                 <div class="media forum-item">
-                                    <a>
-                                        <img
-                                            src="https://bootdey.com/img/Content/avatar/avatar2.png"
-                                            class="mr-3 rounded-circle"
-                                            width="50"
-                                            alt="User"/>
-                                    </a>
-                                    <strong>
+                                    {% if message.sender_username == current_user.username %}
+                                        <a>
+                                            <img
+                                                    src="{{ get_prof_pic(message.recipient_username) }}"
+                                                class="mr-3 rounded-circle"
+                                                width="50"
+                                                alt="User"/>
+                                        </a>
+                                        <strong>
                                         <a href="/" data-toggle="collapse" class="text-body">
-                                            {% if message.sender_username == current_user.username %}
                                                 {{ message.recipient_username }}
-                                            {% else %}
-                                                {{ message.sender_username }}
-                                            {% endif %}
+                                            
                                         </a>
+                                    {% else %}
+                                            <a>
+                                                <img
+                                                    src="{{ get_prof_pic(message.sender_username) }}"
+                                                    class="mr-3 rounded-circle"
+                                                    width="50"
+                                                    alt="User"/>
+                                            </a>
+                                            <strong>
+                                            <a href="/" data-toggle="collapse" class="text-body">
+                                                    {{ message.sender_username }}
+                                                
+                                            </a>
+                                    {% endif %}
                                     </strong>
```

Comment:
have 2 `</strong>` inside each if/else. It's harder to read this way.

Reply:
I take it back. Add `fw-bold` to the class and remove `<strong>` because it's usually bad practice to use those tags.